coinbase/staking-client-library-go
# Switch to preferred AuthN mechanism

## Description Of Change

This PR helps:

* Update BuildJWT logic to handle \n chars that can get added while copy/pasting the api private key into example code.
* Update examples to use apiKeyName and apiPrivateKey instead of reading from a api key file.
* Update `README.md` with new preferred auth mechanism.

## Testing Procedure



1. Visually inspected the README
2. Ran through the quick start guide to make sure it's working
